### ISSUE TITLE
Random Play always hit same songs.

### ISSUE THREAD

**Issue by ShiinaManatsu** [digimezzo/dopamine-windows#934]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report  
[ ] Feature request
</code></pre>

## I'm using these versions

* Windows version:
* Dopamine version:

## Reproduction steps

I use dopamine for a long time with random play, but seemed I always hit the same songs?
Maybe the random method don't have seeds?

**Comment by AhmetFUsta** [comment]
Got this issue as well but **only** if Shuffle Mode is active.
If I deactivate it and hit Shuffle All then it generated a new queue. If I hit next song and then hit Shuffle All again without deactivating Shuffle Mode it generates the same queue or just plays the same one.

**Comment by ShiinaManatsu** [comment]
> Got this issue as well but **only** if Shuffle Mode is active.
> If I deactivate it and hit Shuffle All then it generated a new queue. If I hit next song and then hit Shuffle All again without deactivating Shuffle Mode it generates the same queue or just plays the same one.

Yeah, it's not only that, for a long time use that I found that some songs will be at a high frequency to be played, that's really annoying.
